package.json

```diff
@@ -93,6 +93,7 @@
     "reduxsauce": "^1.0.1",
     "reselect": "^4.0.0",
     "sass": "^1.22.10",
+    "shortid": "^2.2.15",
```

Comment:
Not sure if you guys think this is necessary / worth the additional little dependency. I just use shortid to generate a little unique localStorage key so that in theory if the user had two OAuth login attempts going at once they wouldn't collide.

Reply:
Seems fine to me, I think I've used this in other projects previously.